gpiozero/boards.py

```diff
@@ -1185,3 +1185,86 @@ def on(self):
     def off(self):
         self.value = False
 
+class SevenSegmentDisplay(LEDBoard):
+    layouts = {
+        '1': (False, True, True, False, False, False, False),
+        '2': (True, True, False, True, True, False, True),
+        '3': (True, True, True, True, False, False, True),
+        '4': (False, True, True, False, False, True, True),
+        '5': (True, False, True, True, False, True, True),
+        '6': (True, False, True, True, True, True, True),
+        '7': (True, True, True, False, False, False, False),
+        '8': (True, True, True, True, True, True, True),
+        '9': (True, True, True, True, False, True, True),
+        '0': (True, True, True, True, True, True, False),
+        'A': (True, True, True, False, True, True, True),
+        'B': (False, False, True, True, True, True, True),
+        'C': (True, False, False, True, True, True, False),
+        'D': (False, True, True, True, True, False, True),
+        'E': (True, False, False, True, True, True, True),
+        'F': (True, False, False, False, True, True, True),
+        'G': (True, False, True, True, True, True, False),
+        'H': (False, True, True, False, True, True, True),
+        'I': (False, False, False, False, True, True, False),
+        'J': (False, True, True, True, True, False, False),
+        'K': (True, False, True, False, True, True, True),
+        'L': (False, False, False, True, True, True, False),
+        'M': (True, False, True, False, True, False, False),
+        'N': (True, True, True, False, True, True, False),
+        'O': (True, True, True, True, True, True, False),
```

Comment:
See, that's what I mean about different people wanting to map different characters to different 7-seg combinations... ;-)